src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -1,15 +1,46 @@
 import * as React from 'react';
+import { get } from 'lodash';
+import { EntityContentDivider, EntityContentSection } from '../';
+import ControlledDataTable from '../../../uikit/DataTable/ControlledDataTable';
+import FamilyTable from './Utils/FamilyTable';
+import sanitize from './Utils/sanitize';
 
-import { EntityContentSection, EntityContentDivider } from '../';
+//https://kf-qa.netlify.com/participant/PT_C954K04Y#summary tons of phenotypes
+//https://kf-qa.netlify.com/participant/PT_CB55W43A#clinical family has mother and child being affected
+
+const ParticipantClinical = ({participant}) => {
+
+  const diagHeads = [
+    { Header: 'Diagnosis Category', accessor: 'diagnosis_category' },
+    { Header: "Diagnosis", accessor: "diagnosis"},
+    { Header: "Diagnosis (Mondo)", accessor: "mondo_id_diagnosis" },
+    { Header: 'Diagnosis (NCIT)', accessor: "ncit_id_diagnosis" },
+    { Header: 'Diagnosis (Source Text)', accessor: 'source_text_diagnosis' },
+    { Header: "Age at event", accessor: "age_at_event_days" },
+    { Header: 'Shared with', accessor: "harmonized.alignedreads" }
+  ];
+
+  const diagnoses = get(participant, "diagnoses.hits.edges", []).map(ele => sanitize(get(ele, "node", {})));
+  //const phenotypes = getNodes(participant, "phenotype", []);
 
-const ParticipantClinical = () => {
   return (
     <React.Fragment>
-      <EntityContentSection title="Diagnoses">Diagnoses Chart</EntityContentSection>
+      <EntityContentSection title="Diagnoses">
+        {diagnoses.length === 0 ?
+          <div>No diagnoses.</div> :
+          <ControlledDataTable loading={false} columns={diagHeads} data={diagnoses} dataTotalCount={-1} onFetchData={() => null} />
```

Comment:
Sorry about that, `ControlledDataTable` should have allowed to use it in a "static" way more easily.

You could pass it a `manual={false}` property, and pass that property to the `ReactTable` component. Just be sure to default the value of `manual` inside `ControlledDataTable` to `false`.

Reply:
I'm not sure what you mean :/ I looked at the source of ControlleDataTable and the "fake" props I'm passing it (loading, dataTotalCount and onFetchData) are all required. I feel like if we're adding a `manual` prop we'd have to remove the propTypes that are there right now.

Reply:
You're right, it would need a little more refactor.
Let's forget about this.